### ISSUE TITLE
"AABB size is negative" when importing glb

### ISSUE THREAD

**Issue by mrfisty** [godotengine/godot#69516]
### Godot version

v4.0.beta6.official [7f8ecffa5]

### System information

Windows 11 

### Issue description

Receiving lots of "AABB size is negative" error messages. I've imported a 3D model (glb file) and it's caused the engine to move extremely slow to the point its near unusable when it comes to saving/loading/running. 

The model started in Character Creator 4, then imported to Adobe's Mixamo for rigging, then imported into Blender and exported from that as a glb to Godot (Character Creator 4 -> Mixamo -> Blender -> Godot).

### Steps to reproduce

Here is a download link the the model being used:  [Google Drive](https://drive.google.com/file/d/1d0tSbB-tA7LOvWr5bIG9spoxKEfS2UWw/view?usp=sharing )

You can just import the glb file, but there's a blend file too if anyone wants to see what it looks like before it's exported as a glb.

### Minimal reproduction project

N/A

**Comment by adamscott** [comment]
This doesn't invalidate the issue, but your model is 104.9MB. This is quite huge for a game engine.

**Comment by clayjohn** [comment]
Duplicate of https://github.com/godotengine/godot/issues/56458. 

Closing this issue so we can track it in one place

**Comment by mrfisty** [comment]
Can this thread/issue be re-opened? I've updated Godot to the latest version (Beta 8) and the issue persist even though it seemed it might have been fixed here #65035.

Still getting an infinite amount of "E 0:00:21:0732   merge_with: AABB size is negative, this is not supported. Use AABB.abs() to get an AABB with a positive size." and this seems to be making the engine move very slow (loading/saving/running).
